test/replcompletions.jl

```diff
@@ -457,7 +457,6 @@ try
     end
 finally
     @test pop!(LOAD_PATH) == path
-    rm(path, recursive=true)
```

Comment:
only the first call, before mkpath gets run, and files get created under it. something else was happening in your case, open a separate issue instead of putting it into multiple unrelated pr's please